sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.2.0-beta.1 (Unreleased)
 
+- Updated `api-version` to `2020-06-01`
```

Comment:
Is it `2020-06-30`?

Reply:
The api version of compute is `2020-06-01`. The package is `2020-06-01`.